src/Loaders/Loaders.story.tsx

```diff
@@ -0,0 +1,112 @@
+/** @jsx jsx */
+import { jsx } from "@emotion/core";
+import { storiesOf } from "@storybook/react";
+import { LoadingSpinner, Size } from "./LoadingSpinner";
+import { Button } from "../Button";
+import { colors } from "../colors";
+import * as typography from "../typography";
+import { DemoSection, DemoGroup } from "../shared/DemoSection";
+
+const SPINNER_SIZES: Size[] = ["large", "medium", "small", "tiny", "icon"];
+
+storiesOf("Loaders", module)
+  .addParameters({
+    options: {
+      showPanel: false,
+    },
+  })
+  .add("Catalog", () => (
+    <div css={{ color: colors.black.base }}>
+      <DemoSection
+        title="Spinners"
+        description="Spinners are used when we are unable to determine loading time. Ideally, they should appear as briefly and infrequently as possible."
+      >
+        <DemoGroup
+          title="On Pages & Cards"
+          description="Spinners will most often appear on full pages and cards. Whenever possible, they should be paired with descriptive text that helps the user understand exactly what the system is working toward. Spinners should be center-aligned to the page with any text spaced 20 px below."
+        >
+          <div
+            css={{
+              flex: "1 1 0%",
+              border: `1px solid ${colors.silver.dark}`,
+              borderRadius: 8,
+              margin: 6,
+            }}
+          >
+            {SPINNER_SIZES.map(size => (
+              <div key={size} css={{ margin: 20 }}>
+                <span
+                  css={{
+                    ...typography.base.small,
+                    fontWeight: 600,
+                    display: "block",
+                    textTransform: "uppercase",
+                  }}
+                >
+                  {size}
+                </span>
+                <LoadingSpinner css={{ display: "block" }} size={size} />
+              </div>
+            ))}
+          </div>
+          <div
+            css={{
+              backgroundColor: colors.blilet.darker,
```

Comment:
Q: Why did you chose to use `colors.blilet.darker ` instead of `colors.black.base`?

Reply:
Since I dont know of any screens with black backgrounds, black background didnt seem like a useful thing to have visualized. I have seen components with the `blilet.darker` background color like the sidebar so I thought that might be a more common background and more useful to have visualized

Reply:
Great answer. We're going to have pages with black backgrounds in the near future. 

Check out this quip doc: https://apollographql.quip.com/ZQtPAw13fRcH/Information-Architecture-Nav-Concepts#RMBACASwNB7

Also the background on buttons: https://zpl.io/amdN6Pr

TLDR; Please change the background color to `colors.black.base`

Reply:
Great answer. We're going to have pages with black backgrounds in the near future. 

Check out this quip doc: https://apollographql.quip.com/ZQtPAw13fRcH/Information-Architecture-Nav-Concepts#RMBACASwNB7

Also the background on buttons: https://zpl.io/amdN6Pr

TLDR; Please change the background color to `colors.black.base`